### DISCUSSION TITLE
Does "checkver": "github" depend on semantic versioning?

### DISCUSSION THREAD

**Discussion by superusercode** [ScoopInstaller/Extras#11276] [Q&A] [answered]
I have a PR at https://github.com/Calinou/scoop-games/pull/937 failing checkver. The release itself denotes a patch after the version like https://github.com/yairm210/Unciv/releases/tag/4.6.12-patch1 . Is not following semver exactly why it is failing?

**Comment by StarsbySea** [comment, marked as answer]
Based on the PR you provided, it feels like an issue with the Github RSS feed, and it looks like the issue is resolved, so please consider marking this discussion as resolved yet.
